mlflow/models/model.py

```diff
@@ -191,7 +197,14 @@ def to_dict(self):
 
     def to_yaml(self, stream=None):
         """Write the model as yaml string."""
-        return yaml.safe_dump(self.to_dict(), stream=stream, default_flow_style=False)
+
+        # Do not save mlflow_version attribute if it's not defined
+        info = self.to_dict()
+        mlflow_version_key = "mlflow_version"
```

Comment:
Since this key name is referenced in more than one place(ref: line 243), perhaps we could create a module private constant?
`_MLFLOW_VERSION_KEY = "mlflow_version"` towards the top and reference that throughout the module.

Reply:
+1

Reply:
sure, will do